paymentsheet/src/main/java/com/stripe/android/paymentsheet/ExperimentalPaymentSheetDecouplingApi.kt

```diff
@@ -1,5 +1,9 @@
 package com.stripe.android.paymentsheet
 
+@Deprecated(
+    message = "This feature is no longer experimental. You can remove the annotation.",
+    level = DeprecationLevel.HIDDEN,
```

Comment:
Should this be a `DeprecationLevel.WARNING` instead? I initially wanted to use `HIDDEN` to not expose it to new users, but not sure if we want to _force_ removal for existing beta users.

Reply:
Should we just remove it? Feels acceptable to me.

Reply:
I was thinking the other side of confusion :O Maybe a user sees the annotation is deprecated, and thinks we deprecated the feature!

Reply:
What's the user experience if it's removed?  Will they see a mysterious compiler error?